models/classes/preview/ItemPreviewerService.php

```diff
@@ -35,7 +35,8 @@ class ItemPreviewerService extends ConfigurableService
     const SERVICE_ID = 'taoItems/ItemPreviewer';
     const REGISTRY_ENTRY_KEY = 'taoItems/previewer/factory';
     const PREVIEWERS_KEY = 'previewers';
-    
+    const PLUGINS_KEY = 'plugins';
```

Comment:
```suggestion
    private const PLUGINS_KEY = 'plugins';
```

The other constants' visibility modifiers would be nice to add as well.